.buildkite/scripts/setenv.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -euo pipefail
+
+WORKSPACE="$(pwd)/bin"
+
+add_bin_path(){
+    mkdir -p ${WORKSPACE}
+    export PATH="${WORKSPACE}:${PATH}"
+}
+
+with_go() {
+    mkdir -p ${WORKSPACE}
+    retry 5 curl -sL -o ${WORKSPACE}/gvm "https://github.com/andrewkroh/gvm/releases/download/${SETUP_GVM_VERSION}/gvm-linux-amd64"
+    chmod +x ${WORKSPACE}/gvm
+    eval "$(gvm $(cat .go-version))"
+    go version
+    which go
+    echo "Installing dependencies..."
+    go get github.com/jstemmer/go-junit-report
```

Comment:
Why do we need to get the `go-junit-report`

Reply:
I suppose the `go-junit-report` tool is installed on the Jenkins server, but in the Buildkite pipeline, I got the error https://buildkite.com/elastic/fleet-server/builds/161#0187e582-99f6-48ce-b791-b0b90b0a7d6f in the pipeline.
Probably we can install all dependencies at one time, I saw the step with downloading several dozen necessary components/tools. I thought about it, but we need to be ready for the prod with the pipeline and after that analyze all dependencies again.

Reply:
Looking at the Makefile, we actually dont need to install it here. 
Its done as part of the Makefile target https://github.com/elastic/fleet-server/blob/5cf1e3874c85e4e63d8835aa8461e4709ea53735/Makefile#L162

Reply:
I think you installed it as part of the line 20, the Makefile target installs it (but because it already exists does nothing), but it was not working because the PATH needed adjustment. 
You changed the `PATH` in commit [`9f9e036` (#2543)](https://github.com/elastic/fleet-server/pull/2543/commits/9f9e0361e625f1a1eefc0cff12f7bbdd0e196e09) so I think removing line 20 and having only the Makefile should now work just fine.

Reply:
Yes, you are right, I've commented these lines and everything is OK, the issue was with GOPATH/bin
Will delete them in the next commit